### PR TITLE
Update doc it to remove references of deprecated packages

### PR DESCRIPTION
**Pull request by bkarambe**
Fixes # BLUI-3642

#### Changes proposed in this Pull Request:
- Removed references of deprecated packages from doc it

#### Screenshots:
Screenshot 2022-12-19 at 3 33 59 PM

#### To Test:
- yarn start

### CODE REVIEW

File: src/docs/patterns/visualizations.mdx
Comment (huayunh) [inline src/docs/patterns/visualizations.mdx:124]: you will want to alter the verbiage here too. We don't **provide** themes / utilities anymore but **recommend** them instead

File: src/docs/development/frameworks-mobile/react-native.mdx
Comment (joebochill) [inline src/docs/development/frameworks-mobile/react-native.mdx:28]:
```suggestion
You will be asked to provide a name for your project and whether you want to use the Brightlayer UI eslint and prettier configurations files.
```

File: src/docs/patterns/visualizations.mdx
Comment (joebochill) [inline src/docs/patterns/visualizations.mdx:51]: not sure if this comment syntax works / plays nicely with prettier when using v2 of mdx

**Comment by joebochill** [comment]
Did you make sure to check if there were any internal / external links to the visualization packages and/or github other than on the visualizations page?

File: src/docs/patterns/visualizations.mdx
Comment (huayunh) [inline src/docs/patterns/visualizations.mdx:53]: you can remove this entire TODO... the issue has been deleted

**Comment by bkarambe** [comment]
> Did you make sure to check if there were any internal / external links to the visualization packages and/or github other than on the visualizations page?

Do we need to remove the internal links  from release notes as well?

**Comment by bkarambe** [comment]
> Did you make sure to check if there were any internal / external links to the visualization packages and/or github other than on the visualizations page?

Yes I checked

**Comment by jeffvg** [comment]
not sure if we still ref this file but has entries for mapbox & chartjs
https://github.com/brightlayer-ui/doc-it/blob/2511e4b498bdcd7e5b9b8b4d498d674079e912c5/src/docs/migration.mdx

**Comment by joebochill** [comment]
> 

The repos still exist, so you shouldn't need to remove them from the release notes (I would only remove or update them if there are _broken_ links).

**Comment by joebochill** [comment]
> not sure if we still ref this file but has entries for mapbox & chartjs https://github.com/brightlayer-ui/doc-it/blob/2511e4b498bdcd7e5b9b8b4d498d674079e912c5/src/docs/migration.mdx

We should not need to make any changes to the migration document...that explains how to move from the pxblue scoped packages to the blui ones...the blui scoped packages still exist, so if someone is migration from pxblue, then the instructions are still valid (they'd just be using a still-unsupported package).

**Comment by joebochill** [comment]
Hold off on merging this until after the MUI 5 update has gone into dev

**Comment by joebochill** [comment]
Wait until after @daileytj MUI updates go in before you merge this.

**Review by huayunh — APPROVED** [review]

**Pull request merged**
